.readme-partials/INSTALLATION.md

```diff
@@ -1,13 +1,23 @@
-Clone this repository onto your hard drive and then use Composer to install all dependencies:
+If you normally use WP-CLI on your web host or via Brew, you're most likely using the Phar executable (`wp-cli.phar`). This Phar executable file is the "built", singular version of WP-CLI. It is compiled from a couple dozen repositories in the WP-CLI GitHub organization.
 
-```
-git clone https://github.com/wp-cli/wp-cli-dev
-cd wp-cli-dev
-composer install
-```
-
-This will:
+In order to make code changes to WP-CLI, you'll need to set up this `wp-cli-dev` development environment on your local machine. The setup process will:
 
 1. Clone all relevant packages from the `wp-cli` GitHub organization into the `wp-cli-dev` folder, and
 2. Install all Composer dependencies for a complete `wp-cli-bundle` setup, while symlinking all of the previously cloned packages into the Composer `vendor` folder.
 3. Symlink all folder in `vendor` into corresponding `vendor` folders in each repository, thus making the centralized functionality based on Composer available in each repository subfolder.
+
+First, clone the GitHub repository:
```

Comment:
I like the details you've added. I'm wondering if we could restructure it to keep only one list of steps. When I read the current version, after reading the first bullet I open the terminal and start thinking about how to clone all packages from the organization. It's not fully clear if I should perform steps 1-3 and then follow the steps below, or if the steps below are detailed instructions for steps 1-3. If the latter is true, could we compile them in one list that explains what should be done in a given step and include a sample command?

Reply:
Fair enough, I consolidated to a single code block with 056411e